pkg/acsengine/orchestrators.go

```diff
@@ -0,0 +1,207 @@
+package acsengine
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/acs-engine/pkg/api"
+	"github.com/Azure/acs-engine/pkg/api/common"
+)
+
+// OrchestratorsInfo contains list of release info for supported orchestrators
+type OrchestratorsInfo struct {
+	Orchestrators []*api.OrchestratorInfo `json:"orchestrators"`
```

Comment:
should this be named as orchestratorInfos

Reply:
```
type OrchestratorInfosResult struct {
   OrchestratorInfos []*api.OrchestratorInfo `json:"orchestratorInfos"`
}
```

Reply:
changed